xbmc/Application.cpp

```diff
@@ -1156,6 +1138,14 @@ bool CApplication::Initialize()
   g_curlInterface.Load();
   g_curlInterface.Unload();
 
+  if (g_windowManager.Initialized())
+  {
+    g_windowManager.CreateWindows();
+
+    if (g_advancedSettings.m_splashImage)
+      g_windowManager.ActivateWindow(WINDOW_SPLASH);
```

Comment:
and revert everything you've changed here except the `g_windowManager.Delete(WINDOW_SPLASH);`